### DISCUSSION TITLE
Camera access with avicap32.dll

### DISCUSSION THREAD

**Discussion by Pratham95VP** [AvaloniaUI/Avalonia#14950] [Q&A]
I am using avicap32.dll and user32.dll to access web cam and display video feed. 
```
[DllImport("avicap32.dll")]
protected static extern int capCreateCaptureWindowA([MarshalAs(UnmanagedType.VBByRefStr)] ref string lpszWindowName,
    int dwStyle, int x, int y, int nWidth, int nHeight, int hWndParent, int nID);
        
```
how to set Avalonia.Controls.Image as parent to this window ? [https://github.com/micjahn/ZXing.Net/blob/master/Clients/WindowsFormsDemo/WebCam.cs](url)  
I am referring to this code which is for  Windows Forms. 
I know It will be easier with libraries like Open.CV or Emgu.Cv but i want to try win32API.

**Comment by timunie** [comment]
I'd suggest to look into WriteableBitmap or inherit Control and override OnRender

**Comment by Arafath101** [comment]
Yes what I do

**Reply by Pratham95VP** [reply]
I am not getting what are you trying to indicate ? Can you elaborate please?

**Comment by Bee778** [comment]
![Screenshot_20240403-100843_Chrome](https://github.com/AvaloniaUI/Avalonia/assets/165693594/39bc2c0f-eef4-4692-a68d-7ca012e1f015)

**Comment by Arafath101** [comment]
What should I want to do tell me properly

**Comment by Bee778** [comment]
You so funny

**Reply by Pratham95VP** [reply]
?

**Comment by timunie** [comment]
@Pratham95VP it was just a spam bot I guess 🤷

**Reply by Pratham95VP** [reply]
ok
